bc/channel/models.py

```diff
@@ -131,6 +138,8 @@ def self_url(self):
                 return f"{instance_part}@{account_part}"
             case self.BLUESKY:
                 return f"https://bsky.app/profile/{self.account_id}"
+            case self.THREADS:
+                return f"https://www.threads.net/@{self.account_id}"
```

Comment:
Let's use `account` instead of `account_id`. After generating my token using the `get-threads-keys.py` script, I noticed the suggested value for `account_id` is simply a number.

```suggestion
                return f"https://www.threads.net/@{self.account}"
```